lib/elixir/lib/protocol.ex

```diff
@@ -448,7 +448,7 @@ defmodule Protocol do
   ## Examples
 
       # Get Elixir's ebin directory path and retrieve all protocols
-      iex> path = :code.lib_dir(:elixir, :ebin)
+      iex> path = Path.join(:code.lib_dir(:elixir), "ebin")
```

Comment:
We should use `Application.app_dir` :) here and everywhere else :D

Reply:
Looks much nicer indeed! [fc6d026](https://github.com/elixir-lang/elixir/pull/13395/commits/fc6d02639a114cf15fe6b3f29d1d627b24c8907d)